### DISCUSSION TITLE
Ability to transfer content to different users

### DISCUSSION THREAD

**Discussion by knynkwl** [craftcms/cms#10113] [Ideas]
Right now, you have to delete the user in order to transfer content. It would be great if you could transfer all content without deleting a user.

**Comment by mijewe** [comment]
Ah so this is why they have the "I have done a search for similar discussions" checkbox...

---

When you go to delete a user, Craft gives you the option of transferring Old User's content to New User.

However in a Commerce site, you cannot delete a User that has any Orders attached to them.

So it would be nice to be able to transfer content from Old User to New User without having to delete Old User.

---

My use case here is that I'm building an membership site where the client is worried about multiple users from the same organisation signing up, and the client wants to be able to merge users together when this happens.

So if both users have made purchases, this wouldn't be possible. Or rather, the admin would have to manually reassign all entries from User B to User A and then suspend User B. Which is fine but a bit messy.

**Reply by brandonkelly** [reply]
Yeah we agree this should be possible :)
